cmd/ocm-backplane/login/login.go

```diff
@@ -296,6 +296,10 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 			return fmt.Errorf("cannot connect to backplane API URL, check if you need to use a proxy/VPN to access backplane: %v", connErr)
 		}
 
+		// Add suggestion to run connectivity health check if login fails
```

Comment:
Can we use `logger` here instead of `fmt.Print` so that we can adjust the level easily?